### PR TITLE
Regularly update multiple tags

### LINKED ISSUE #128: Outdated variants?
Hi,

As per a comment on a previous issue, images should be getting updates on the first Saturday of each month. However, looking at the PR that introduced the dynamic github actions runner fetcher, it seems like at least `ghcr.io/cirruslabs/macos-sonoma-xcode:15.1` has not been updated to reflect that change.

Is this schedule still applicable? 

Many thanks 🙏

### PR DESCRIPTION
**Pull request by fkorotkov**
Update the last two version of macOS and the last two versions of Xcode for each of macOS versions.

Fixed #128

### CODE REVIEW

**Comment by fkorotkov** [comment]
This change also removes pushes to ECR which proved to be very flaky and unreliable. FYI @riain0.

File: .cirrus.yml [hunk 1]
```diff
@@ -28,45 +32,61 @@ task:
...
   always:
     cleanup_script:
-      - tart delete sonoma-vanilla || true
+      - tart delete sonoma-vanilla
 
 task:
-  name: "Update Non-Vanilla Images"
+  name: "Update Base Images ($MACOS_VERSION)"
+  alias: update-base
+  env:
```
[the remark this answers is not in the document]
Comment (fkorotkov) [reply .cirrus.yml:40]: wow! Great catch! Fixed in [c4fe4bc](https://github.com/cirruslabs/macos-image-templates/pull/130/commits/c4fe4bcb6da8e71e6080e522ebcd6c9803502d4c)

**Comment by edigaryev** [comment]
Do we still need the `Release Xcode $CIRRUS_TAG` task?

It seems that the new `Update Xcode Images ($MACOS_VERSION $XCODE_VERSION)` tasks push the same images as the task above.

**Comment by fkorotkov** [comment]
I left it so we can release beta versions or quickly new release versions.

**Review by edigaryev — APPROVED** [review]

**Pull request merged**